pkg/server/internal/endpoint/bgpconf.go

```diff
@@ -0,0 +1,66 @@
+package endpoint
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/openelb/openelb/api/v1alpha2"
+	"github.com/openelb/openelb/pkg/server/internal/lib"
+	"github.com/openelb/openelb/pkg/server/internal/service"
+)
+
+type bgpConfEndpoints struct {
+	service service.BgpConfService
+}
+
+func (b *bgpConfEndpoints) Register(r chi.Router) {
+	r.Post("/bgpconf", b.create)
+	r.Get("/bgpconf", b.get)
+	r.Delete("/bgpconf", b.delete)
```

Comment:
Wouldn't it be better if we set a uniform url prefix?
```
http://host:port/apis/v1/bgpconf
http://host:port/apis/v1//bgppeer
```

Reply:
> apis/v1/bgpconf
apis/v1/bgppeer

Yeah I agree that we can improve the routes like this.

Btw as OpenELB has both v1 and v2 at this moment, how are we supposed to tackle this situation? Should create separate routes for both v1 and v2?

Reply:
> > apis/v1/bgpconf
> > apis/v1/bgppeer
> 
> Yeah I agree that we can improve the routes like this.
> 
> Btw as OpenELB has both v1 and v2 at this moment, how are we supposed to tackle this situation? Should create separate routes for both v1 and v2?

Just provide the url of v1alpha2.